Makefile

```diff
@@ -407,10 +407,10 @@ proto-lint:
 proto-check-breaking:
 	@$(DOCKER_BUF) breaking --against $(HTTPS_GIT)#branch=main
 
-TM_URL              = https://raw.githubusercontent.com/tendermint/tendermint/v0.34.19/proto/tendermint
+TM_URL              = https://raw.githubusercontent.com/tendermint/tendermint/v0.34.20/proto/tendermint
 GOGO_PROTO_URL      = https://raw.githubusercontent.com/regen-network/protobuf/cosmos
-CONFIO_URL          = https://raw.githubusercontent.com/confio/ics23/v0.6.3
-SDK_PROTO_URL 	    = https://raw.githubusercontent.com/cosmos/cosmos-sdk/v0.45.6/proto/cosmos
+CONFIO_URL          = https://raw.githubusercontent.com/confio/ics23/014bd93b66bb57e5f250be0c9a344505f7d0fa70
```

Comment:
Is there release version of `confio/ics23` that we can use here?
I think it would best to avoid using a commit hash

Reply:
cc @ethanfrey

Reply:
Looks like 0.7.1 is the latest stable tag https://github.com/confio/ics23/tags

Reply:
sadly, it didn't work -- the correct release though, is v0.7.1

(no clue why it wouldn't work)